### ISSUE TITLE
Nintendo 3DS port

### ISSUE THREAD

**Issue by Krzycho666** [Gigoteur/PX8#8]
Would love to see it on 3DS. =]

**Comment by hallucino** [comment]
Have you any example of rust program on 3DS ?

**Comment by Krzycho666** [comment]
Is that enough? =]
https://github.com/rust3ds

**Comment by hallucino** [comment]
Do you want to try ? I don't have a Nintendo :/

**Comment by Krzycho666** [comment]
I do have, but I've never done anything using Rust. I've asked about that, because your project is the only open source one of this kind, and it supports PICO-8 roms - which are the most popular out there. And on the other hand, 3DS feets to this kind of project as hell!

**Comment by hallucino** [comment]
any volunteers ? :)

**Comment by mar-sw** [comment]
I would try to do it, I have a 3DS with CFW, so...
:)

**Comment by Krzycho666** [comment]
I can help, but not sooner then a month from now.

**Comment by mar-sw** [comment]
Yes, no problem! :)

2017-09-12 16:13 GMT-03:00 Hallucino <notifications@github.com>:

**Comment by hallucino** [comment]
@mar-sw have you try ?

**Comment by Ferk** [comment]
What's the status of the libretro backend? Maybe it'd be easier to compile a libretro core for 3ds, since retroarch already offers a 3DS port (and many other consoles too).

**Comment by hallucino** [comment]
Yep I agree it can be a good idea. We need few things for working but I did few tests and right now it works

**Comment by alice902** [comment]
How's going? I'd love to play it on my 3dz.
